docs/reverse_proxy.md

```diff
@@ -76,6 +76,17 @@ canonicalise/normalise the URI.
           }
         }
 
+### Caddy 2
+
+        matrix.example.com {
+          reverse_proxy /_matrix/* http://localhost:8008
+          reverse_proxy /.well-known/matrix/* http://localhost:8008
```

Comment:
none of the other examples suggest forwarding .well-known - there may be valid reasons to do so, but having the examples out of sync sounds confusing to me.

Reply:
```suggestion
```

I also don't think matrix servers will resolve these requests.

Reply:
I did wonder if I should include this line, but it look me a while to figure out so I kept it. Matrix does resolve it, an example output looks like this:

```
$ curl localhost:8008/.well-known/matrix/client
{"m.homeserver": {"base_url": "https://<redacted>"}}
```

It seems like to me the rest of the examples should be updated to match this one, but I can remove it if you still think immediate consistency is more important.

Reply:
> if you still think immediate consistency is more important.

I do, really.

Reply:
Huh, TIL!

Reply:
Ok, removed!